binder/start

```diff
@@ -1,6 +1,21 @@
 #!/bin/bash
-DIR=$( cd "$(dirname "$0")" ; pwd -P )
 
-[ ! -e /tmp/supervisor.sock ] && supervisord -c $DIR/../tutorials/supervisor/supervisord.conf
+[ ! -e /tmp/supervisor.sock ] && supervisord -c supervisor/supervisord.conf
```

Comment:
After all that, the `$DIR` stuff wasn't buying us anything here.  Seems to me that this is simpler to maintain.